dev/ComboBox/ComboBoxAttachedProperties.idl

```diff
@@ -0,0 +1,16 @@
+﻿namespace MU_XCP_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[default_interface]
+runtimeclass ComboBoxAttachedProperties
+{
```

Comment:
What is actually setting this Attached Property? I don't see it referenced anywhere?

Reply:
Never mind. I see it now.

---
In reply to: [301824243](https://github.com/microsoft/microsoft-ui-xaml/pull/1023#discussion_r301824243) [](ancestors = 301824243)